javascript/node/streams/piping-readable-streams.md

```diff
@@ -58,30 +44,30 @@ The `pipe()` function is used in the above example to write the content of `file
 How would you pipe the data from a readStream of `fileA.txt` to a writeStream of `fileB.txt`, both created through the `fs` module? Basically you have to move all the data from `fileA` to `fileB`.
 
 ```javascript
-var fs = require('fs')
+let fs = require('fs')
 
-var r = ???(???)
-var w = ???.???(???)
+let r = ???(???)
+let w = ???.???(???)
 
 ???.???(???)
 ```
 
-* fs.createReadStream
-* 'fileA.txt'
-* fs
-* createWriteStream
-* 'fileB.txt'
-* r
-* pipe
-* w
-* fs.readStream
-* readStream
-* writeStream
-* fs.writeStream
-* transfer
-* move
-* fileA
-* fileB
+* `fs.createReadStream`
+* `'fileA.txt'`
+* `fs`
+* `createWriteStream`
+* `'fileB.txt'`
+* `r`
+* `pipe`
+* `w`
+* `fs.readStream`
+* `readStream`
+* `writeStream`
+* `fs.writeStream`
+* `transfer`
+* `move`
+* `fileA`
+* `fileB`
```

Comment:
is there any benefit to enclosing these in backticks? both 
```
* `this`
* this
```
 will display the same in the app

Reply:
Nope, I just did it cause I forgot if it renders properly in the app or not and was lazy to test it 😆

Reply:
perfect then :smile: